modules/HomePage/components/Posts/Posts.tsx

```diff
@@ -10,14 +10,23 @@ import { EStatusCode } from '@modules/shared/api/EStatusCode';
 import { logoutUser } from 'context/AuthUserContext/api/authApi';
 import { addOneVote } from '@modules/HomePage/api/votesApi/voteApi';
 import type { IVotesApi } from '@modules/HomePage/api/votesApi/IvotesApi';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { getPosts } from '@modules/shared/api/getPosts.api';
+import { getUserToken } from '@modules/shared/logic/userAuth/userAuth';
 import { transformPostsMedia, updateVotedPost } from './PostsHelpers';
 import styles from '../../pages/home-page.module.css';
 
 const Posts: FC<IPostFeed.IPosts> = ({ data }): ReactElement => {
   const [posts, setPosts] = useState<IPostFeed.IPost[]>(data.posts);
+  const [hasMore, setHasmore] = useState<boolean>(true);
+  const [postsCount, setPostsCount] = useState<number>(data.postsCount);
   const { redirectToLoginPage } = useRedirect();
   const toastId = useRef<ReactText>();
 
+  useEffect(() => {
+    setHasmore(posts.length < postsCount + 100);
```

Comment:
Why `postsCount + 100` here?

Reply:
to ensure that the feature works fine until the BE implement the limit and the offset